.github/workflows/README.md

```diff
@@ -26,6 +26,23 @@ Format:
     - trigger: issue [opened, labeled], pull-request [opened, labeled]
     - job: assign_to_board
         - Assigns issues and PRs to either Content Development board or BSSw Internal board based on labels ("content: *" or "scope: site-internal")
+* notify-external-contrib.yml (Notify external contributions)
+    - triggers issue [opened], discussion [opened], pull_request [opened, re-opened, review_requested, ready_for_review]
+    - job: notify-external-contributions
+        - Label and send email to bssw-editorial-list for externally opened issues, prs and discussions
+    - **Notes**: Requires a few special items
+        - A personal access token (PAT). A GitHub user creates in the `Developer settings` subsection of your own, personal `Settings`.
+           - This is needed to read the contents (GitHub user names) of the `bssw-editorial-board` team. This could be avoided entirely if we just included the names of bssw editorial board members directly in the action's `.yml` file. Honestly, there probably isn't anything wrong with that because all of our activities for bssw are *public* on GitHub anyways.
```

Comment:
But we don't want to duplicate the list of members in the `bssw-editorial-board` team so some work up front that avoids that duplication was worth the effort (if we expect this site to live for a long time).

Reply:
```suggestion
           - This is needed to read the contents (GitHub user names) of the `bssw-editorial-board` team.
              This could be avoided entirely if we just included the names of bssw editorial board members directly in the action's `.yml` file.
              Honestly, the only thing wrong with that is that it creates a second place to go edit when bssw editorial team membership changes.
```